tests/unit/select_test.coffee

```diff
@@ -344,3 +344,16 @@ test 'optionValuePath with nested valuePath', ->
 
   equal obj2, select.get('selection'),
     'The right selection is retrieved'
+
+test 'shouldEnsureVisible controls whether to ensure visibility', ->
+  expect 1
+
+  select = @subject
+    content: ['foo']
+    shouldEnsureVisible: no
```

Comment:
you may want to test the default case first, where `shouldEnsureVisible` true, check the spy there. And after that, you set this to No, set highlighted, and check the spy again.